swiftshield-Sources/SwiftAttribute.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  SwiftAttribute
+//  swiftshield
+//
+//  Created by Weidian on 22/12/19.
+//  Copyright © 2019 Bruno Rocha. All rights reserved.
+//
+
+import Foundation
+
+enum SwiftAttribute: String {
+    case `public` = "source.decl.attribute.public"
+    case `private` = "source.decl.attribute.private"
+    case `fileprivate` = "source.decl.attribute.fileprivate"
+    case `internal` = "source.decl.attribute.internal"
```

Comment:
What about `open`?

Reply:
I found a reference of above, but there is no reference of open. I will do some testing on open, and exclude open also.